src/API/Google/AdsCampaign.php

```diff
@@ -99,15 +99,16 @@ public function __construct( GoogleAdsClient $client, AdsCampaignBudget $budget,
 	/**
 	 * Returns a list of campaigns with targeted locations retrieved from campaign criterion.
 	 *
-	 * @param bool $exclude_removed Exclude removed campaigns (default true).
-	 * @param bool $fetch_criterion Combine the campaign data with criterion data (default true).
+	 * @param bool  $exclude_removed Exclude removed campaigns (default true).
+	 * @param bool  $fetch_criterion Combine the campaign data with criterion data (default true).
+	 * @param array $args Additional arguments.
```

Comment:
WDYT of being more concrete here?

Maybe 

 @param array $args Arguments for the Ads Campaign Query.